src/github.com/getlantern/flashlight/client/chained.go

```diff
@@ -14,10 +14,12 @@ import (
 	"github.com/getlantern/withtimeout"
 )
 
-// Close connections idle for a period to avoid dangling connections.
-// 1 hour is long enough to avoid interrupt normal connections but short enough
-// to eliminate "too many open files" error.
-var idleTimeout = 1 * time.Hour
+// Close connections idle for a period to avoid dangling connections. 45 seconds
+// is long enough to avoid interrupt normal connections but shorter than the
+// idle timeout on the server to avoid running into closed connection problems.
+// 45 seconds is also longer than the MaxIdleTime on our http.Transport, so it
+// doesn't interfere with that.
+var idleTimeout = 45 * time.Second
```

Comment:
Now that we have an idletiming that returns io.EOF for pending reads and writes instead of "Use of closed network connection", we can make this much shorter.  We make it shorter than the server's idle timeout so that we don't risk sending data on a connection that will subsequently get idled by the server.

I increased the server's idle timeout to 70 seconds in [this commit](https://github.com/getlantern/http-proxy-lantern/commit/0ea7f958212068e2b579860e4e40be9a26a10a40).